internal/pkg/workspace/workspace.go

```diff
@@ -427,9 +458,16 @@ func (ws *Workspace) ReadAddonsDir(svcName string) ([]string, error) {
 	return names, nil
 }
 
-// ReadAddon returns the contents of a file under the service's "addons/" directory.
-func (ws *Workspace) ReadAddon(svc, fname string) ([]byte, error) {
-	return ws.read(svc, addonsDirName, fname)
+// ReadFile returns the content of a file.
+func (ws *Workspace) ReadFile(fPath string) ([]byte, error) {
+	exist, err := ws.fs.Exists(fPath)
+	if err != nil {
+		return nil, fmt.Errorf("check if manifest file %s exists: %w", fPath, err)
```

Comment:
should probably update this error since it could be reading a manifest file or an addon file now